json-spec/extensions/stac-eo-spec.md

```diff
@@ -0,0 +1,60 @@
+# STAC EO Extension JSON Spec
+
+This document explains the fields of the STAC Earth Observation (EO) Extension to a STAC `Item`. EO data is considered to be data that represents a snapshot of the earth for a single date and time. It could consist of multiple spectral bands in any part of the electromagnetic spectrum. Examples of EO data include sensors with visible, short-wave and mid-wave IR bands (e.g., the OLI instrument on Landsat-8), long-wave IR bands (e.g. TIRS aboard Landsat-8), as well as SAR instruments (e.g. Sentinel-1).
+
+## EO Extension Description
+These are the additional fields to appear under a STAC `Item`, as well as the items's assets field and `Collection` level metadata.
+
+### `Item` additions
+| element             | type info                 | name                    | description                                                                                 | 
+|----------------------|---------------------------|-------------------------|---------------------------------------------------------------------------------------------| 
+| eo:collection             | string                      | Collection Name       | The name of the STAC `Collection` this `Item` belongs to |
```

Comment:
Is collection required?  I'm not really understanding what a collection is.

Reply:
I'm leaning towards 'optional' until we really sort out its meaning and how people practically use it. And yeah, it seems like it needs a bit more explanation.

Reply:
I think it would be more appropriate as optional as well. For flat catalogs, we have the concept of a `root` link that would point to the top-level catalog.json file.